tests/cli.cmake

```diff
@@ -56,24 +58,31 @@ execute(10 ${EXE_DIR}/pmdk-convert --from 1.0 --to 1.1) # NO_POOL
 
 file(WRITE ${DIR}/not_a_pool "This is not a pool\n")
 execute(11 ${EXE_DIR}/pmdk-convert ${DIR}/not_a_pool) # POOL_DETECTION
-execute(15 ${EXE_DIR}/pmdk-convert ${DIR}/not_a_pool --from ${VER} -X fail-safety) # CONVERT_FAILED
+if(LEN GREATER 1) # below test case requires more than one supported versions
```

Comment:
LEN is a really bad name for global variable; if you would name it correctly this comment wouldn't be necessary